packages/experiments/src/components/Tile/Tile.scss

```diff
@@ -126,7 +126,7 @@
 .foreground {
   display: inline-flex;
   position: relative;
-  overflow: hidden;
+  overflow: visible;
```

Comment:
@ThomasMichon This is the only one i'm not sure what the background on hidden overflow was;

If you're good, please merge and make sure it is cherrypicked into master.